dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/values/CookiePartitioned.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package com.ibm.ws.http.channel.internal.values;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.http.channel.internal.HttpMessages;
+import com.ibm.wsspi.genericbnf.HeaderKeys;
+import com.ibm.wsspi.http.HttpCookie;
+import com.ibm.wsspi.http.channel.values.HttpHeaderKeys;
+
+public class CookiePartitioned extends CookieData {
+
+    /** Trace component for debugging */
+    private static final TraceComponent tc = Tr.register(CookieSecureData.class, HttpMessages.HTTP_TRACE_NAME, HttpMessages.HTTP_BUNDLE);
```

Comment:
We should be using CookiePartitioned.class here unless I'm missing something.

Also should we stay consistent and do `CookiePartitionedData` like we did with `CookieSameSiteData`?

Reply:
Copy Paste error -- fixing

Reply:
I see the naming convention is a bit all over some CookieX and other CookieXData, just planning to stick with CookiePartitioned?

Reply:
Yes, at this point, I'd rather stick with CookiePartitioned. We can refactor this code (and related classes) later on.